recipes/ribasim/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "ribasim" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cbcce049ef8f88e570932e980e0add5f1048c646a396f60279fa7ca1f85fe744
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=61
+    - pip
+  run:
+    - python >=3.9
+    - geopandas
+    - matplotlib-base
+    - pandas <2.1
+    - pandera !=0.16.0
+    - pyarrow
+    - pydantic =1
```

Metadata says it should be:

```suggestion
    - pydantic >=1.0,<2.dev0
```

Can you confirm?

---

Can confirm, thanks. Our upcoming release will support pydantic 2.

---

No, I had that originally and it couldn't find a suitable dependency. I think the converter from setuptools metadata has a subtle bug, since it works fine when installing from pypi.